class/defaults.yml

```diff
@@ -1,3 +1,16 @@
 parameters:
   ingress_nginx:
     namespace: syn-ingress-nginx
+    release_name: ingress-nginx
+    charts:
+      ingress-nginx: '3.22.0'
+    helm_values:
+      controller:
+        publishService:
```

Comment:
I think it is valid having publishService enabled by default, as Project Syn components are intended to be used for managed services, where the service is managed by the controller anyway.